tools/cldr-apps/src/main/webapp/myvotes.jsp

```diff
@@ -52,9 +52,7 @@ showRecent('submitItems',null,'<%= user %>');
 ...
 
 <script>
-<% if (SurveyTool.USE_DOJO) %>
 showAllItems('allMyItems','<%= user %>');
-<% else %>
 cldrSurvey.showAllItems('allMyItems','<%= user %>');
```

Comment:
Oops, I should have deleted the "cldrSurvey.showAllItems..." line; myvotes.jsp is unused if !USE_DOJO